### PR TITLE
fix(doc): fix the dev local port

### PR DESCRIPTION
**Pull request by CzyerChen**
The default UI address is `http://localhost:3000`.

```
  VITE v4.5.2  ready in 2551 ms

  ➜  Local:   http://localhost:3000/
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,15 @@ npm install
...
-* Send `Request to join SkyWalking slack` mail to the mail list(`dev@skywalking.apache.org`), we will invite you in.
-* For Chinese speaker, send `[CN] Request to join SkyWalking slack` mail to the mail list(`dev@skywalking.apache.org`), we will invite you in.
-* Twitter, [ASFSkyWalking](https://twitter.com/AsfSkyWalking)
-* [bilibili B站 视频](https://space.bilibili.com/390683219)
+
+- Mail list: **dev@skywalking.apache.org**. Mail to `dev-subscribe@skywalking.apache.org`, follow the reply to subscribe to the mail list.
+- Send `Request to join SkyWalking slack` mail to the mail list(`dev@skywalking.apache.org`), we will invite you in.
+- For Chinese speaker, send `[CN] Request to join SkyWalking slack` mail to the mail list(`dev@skywalking.apache.org`), we will invite you in.
+- Twitter, [ASFSkyWalking](https://twitter.com/AsfSkyWalking)
+- [bilibili B 站 视频](https://space.bilibili.com/390683219)
```
Comment (wu-sheng) [inline README.md:45]: Please don't reformat irrelevant codes.

File: README.md [hunk 2]
```diff
@@ -34,14 +34,15 @@ npm install
 npm run dev
 ```
 
-The default UI address is `http://localhost:8080`.
+The default UI address is `http://localhost:3000`.
```
Comment (wu-sheng) [inline README.md:37]: @Fine0830 Please confirm. Should we get this on 8080 to align with webspp deployment mode.

Comment (Fine0830) [reply README.md:37]: It's okay that the port is `3000` in `dev` mode.

**Comment by wu-sheng** [comment]
@CzyerChen Please revert irrelevant parts, the doc fix is good.

**Comment by CzyerChen** [comment]
> @CzyerChen Please revert irrelevant parts, the doc fix is good.

reformatted by `prettier --write` automatically when committing, and can not revert.

image

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**